fake.py

```diff
@@ -297,10 +297,26 @@ def compose_frame(self, frame):
         if self.no_background is False:
             background_frame = next(self.images["background"])
         else:
-            background_frame = cv2.blur(frame,
-                                        (self.background_blur,
-                                         self.background_blur),
-                                        cv2.BORDER_DEFAULT)
+
+            # copy frame by downscaling 
+            frameWidth = frame.shape[1]
+            frameHeight = frame.shape[0]
+
+            width = int(frameWidth * .25)
+            height = int(frameHeight * .25)
+            background_frame = cv2.resize(frame, (width, height), interpolation = cv2.INTER_LINEAR)
+
+            # apply background blur to resized frame
+            for i in range(1, self.background_blur, 2):
+                background_frame = cv2.GaussianBlur(background_frame,
+                    (i,i),
+                    cv2.BORDER_DEFAULT)
+                background_frame = cv2.GaussianBlur(background_frame,
+                    (i,i),
+                    cv2.BORDER_DEFAULT)
```

Comment:
In these function calls `cv2.BORDER_DEFAULT` is being passed as the `sigmaX` parameter, not the `borderType`.

Reply:
Yes, you are right, well spotted.